### DISCUSSION TITLE
Chip and Chip Group aren't working properly

### DISCUSSION THREAD

**Discussion by Virock** [unoplatform/uno#15040] [Q&A]
I'm trying to place a chip with a remove button in it. Here's my code:
```
<utu:ChipGroup SelectionMode="None" CanRemove="True" Margin="0,10,0,0">
  <utu:Chip Content="Default" CanRemove="True" />
</utu:ChipGroup>
```
But it renders like this 
![image](https://github.com/unoplatform/uno/assets/31706335/0ddaef2c-0a15-481d-a8f7-db15ee69c4d8)

In the uno gallery, I see that I'm supposed to add this style: `Style="{StaticResource FilterChipStyle}"`
But when I add it to the code, it doesn't build any more. I get some error saying it can't find the key value FilterChipStyle.

What do I do?

**Comment by kazo0** [comment]
Hi @Virock,

It looks like you do not have the [Material Toolkit theme](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/material-getting-started.html) setup 

You can follow the docs to setup your app with the Material Toolkit package and here is an example of what your code looks like when running in the Uno Playground: http://playground.platform.uno/#469ef2e5

Let me know if you have any issues getting the Material Toolkit theme up and running!

**Reply by Virock** [reply]
Thank you, Sir. I followed the guide and successfully installed the Material theme. The problem is my entire app now has the material theme.
I've scoured the internet but I can't find a way to make the entire app use the Fluent theme and have only the chip element use the material theme.
Is this even possible?
